charmcraft.yaml

```diff
@@ -1,85 +1,67 @@
-# This file configures Charmcraft.
-# See https://juju.is/docs/sdk/charmcraft-config for guidance.
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
 
-# (Required)
-# The charm package name, no spaces
-# See https://juju.is/docs/sdk/naming#heading--naming-charms for guidance.
 name: velero-operator
-
-
-# (Required)
-# The charm type, either 'charm' or 'bundle'.
 type: charm
+title: Velero Operator Charm
 
+summary: Server Charm for backing up and restoring Kubernetes cluster resources using Velero.
 
-# (Recommended)
-title: Charm Template
-
-
-# (Required)
-summary: A very short one-line summary of the charm.
-
-
-# (Required)
 description: |
-  A single sentence that says what the charm is, concisely and memorably.
+  The Velero Charm enables automated backup, restore, and migration of Juju-managed Kubernetes clusters using Velero (https://velero.io).
+  This charm simplifies the deployment and configuration of Velero, integrating seamlessly with cloud storage providers for secure and scalable disaster recovery.
 
-  A paragraph of one to three short sentences, that describe what the charm does.
+links:
+  issues:
+  - https://github.com/canonical/velero-operator/issues
+  source:
+  - https://github.com/canonical/velero-operator
+  website:
+  - https://github.com/canonical/velero-operator
+  - https://velero.io
 
-  A third paragraph that explains what need the charm meets.
+platforms:
+  ubuntu@24.04:amd64:
 
```

Comment:
would be good to add `assumes: k8s-api` to prevent this k8s charm being deployed on "machine" substrate https://canonical-charmcraft.readthedocs-hosted.com/en/stable/reference/files/charmcraft-yaml-file/#assumes

Reply:
That's a nice suggestion!

Added https://github.com/canonical/velero-operator/pull/6/commits/5ad4ad9d7ee08017f0e7b03c134e3e1aaf734908 to address this.

I'll resolve this conversation, just this once @chanchiwai-ray without waiting for an explicit ACK since it's a small one and we want to send the next PR ASAP.